src/main/java/edu/ksu/canvas/model/assignment/Progress.java

```diff
@@ -0,0 +1,108 @@
+package edu.ksu.canvas.model.assignment;
```

Comment:
This Process object is not specific to assignments. It is used by all asynchronous API calls. So it should not be in the `assignment` sub-package.